static/js/script.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function(){
+    $( "#accordion" ).accordion();
+});
```

Comment:
Can you just check this jQuery code against an online Beautify tool?
Some spaces where they shouldn't be, and missing spaces where there should be.
Good example: https://beautifier.io/